Sources/Components/Consent/ConsentTransparency/ConsentTransparencyView/Demo/ConsentTransparencyDemoView.swift

```diff
@@ -41,14 +41,10 @@ public class ConsentTransparencyDemoView: UIView {
         shadedBackgroundView.fillInSuperview()
 
         NSLayoutConstraint.activate([
-            consentTransparencyView.topAnchor.constraint(greaterThanOrEqualTo: shadedBackgroundView.topAnchor, constant: .largeSpacing),
-            consentTransparencyView.bottomAnchor.constraint(lessThanOrEqualTo: shadedBackgroundView.bottomAnchor, constant: -.largeSpacing),
-            consentTransparencyView.leadingAnchor.constraint(greaterThanOrEqualTo: shadedBackgroundView.leadingAnchor, constant: .largeSpacing),
-            consentTransparencyView.trailingAnchor.constraint(lessThanOrEqualTo: shadedBackgroundView.trailingAnchor, constant: -.largeSpacing),
             consentTransparencyView.centerXAnchor.constraint(equalTo: shadedBackgroundView.centerXAnchor),
             consentTransparencyView.centerYAnchor.constraint(equalTo: shadedBackgroundView.centerYAnchor),
-            consentTransparencyView.widthAnchor.constraint(equalToConstant: maxScreenSize.width),
             consentTransparencyView.heightAnchor.constraint(equalToConstant: maxScreenSize.height),
+            consentTransparencyView.widthAnchor.constraint(equalToConstant: maxScreenSize.width),
```

Comment:
I was thinking in other alternatives but this was the best one I was able to come up with. Might be good to consider that we don't support landscape in the iPhone and that most of our users have the newer devices.

Reply:
I tried making the view more dynamic and not at a fixed height, but once I introduced the scroll view it proved difficult in the timeframe I had. I still think it's good practice to make views as dynamic as possible even if we don't support landscape.

Reply:
I think we could look into how to make it more dynamic laterce, the current design doesn't look very good on taercreens.

Reply:
I agree that having it more dynamic is a good practice and we should aim for it, I think that's something we could look into later, especially since right now we are using hardcoded text, but I think it's important to tweak the current design since it doesn't look very good on taller screens.

Reply:
Yes, I agree. This was a compromise since I didn't have time to fiddle with the visuals too much.